en/sensor/px4flow.md

```diff
@@ -0,0 +1,456 @@
+IMAGE_WIDTH     | 64       | R       | Image width (pixels)
+IMAGE_HEIGHT    | 64       | R       | Image height (pixels)
+IMAGE_L_LIGHT   | 0        | RW      | 1: Image sensor low light mode ON, 0: OFF
+IMAGE_NOISE_C   | 1        | RW      | 1: Image sensor noise correction ON, 0: OFF
+IMAGE_TEST_PAT  | 0        | RW      | 1: Gray-shaded test pattern mode ON, 0: OFF
+LENS_FOCAL_LEN  | 16       | RW      | Focal length of lens (mm) |
+POSITION        | 0        | RW      | 0: Only position 0 is used (Bottom: 0, Front: 1, Top: 2, Back: 3, Right: 4, Left: 5)
+SONAR_FILTERED  | 0        | RW      | 1: Kalman filter on sonar output is ON, 0: OFF
+SONAR_KAL_L1    | 0.8461   | RW      | Sonar Kalman gain L1
+SONAR_KAL_L2    | 6.2034   | RW      | Sonar Kalman gain L2
+SYS_ID          | 81       | RW      | [MAVLink](https://mavlink.io/en/) System ID
+SYS_COMP_ID     | 50       | RW      | [MAVLink](https://mavlink.io/en/) Component ID
+SYS_SENSOR_ID   | 77       | RW      | [MAVLink](https://mavlink.io/en/) Sensor ID
+SYS_TYPE        | 0        | RW      | [MAVLink](https://mavlink.io/en/) System Type
+SYS_AP_TYPE     | 1        | RW      | [MAVLink](https://mavlink.io/en/) Autopilot Type
+SYS_SW_VER      | 13XX     | R       | Software Version
+SYS_SEND_STATE  | 1        | RW      | 1: Send [MAVLink](https://mavlink.io/en/) Heartbeat, 0: Not
+USART_2_BAUD    | 115200   | R       | Baudrate USART 2
+USART_3_BAUD    | 115200   | R       | Baudrate USART 3 (Data Output)
+USB_SEND_VIDEO  | 1        | RW      | 1: Send video over USB, 0: Not
+USB_SEND_FLOW   | 1        | RW      | 1: Send flow over USB, 0: Not
+USB_SEND_GYRO   | 1        | RW      | 1: Send gyro data over USB, 0: Not
+USB_SEND_FWD    | 0        | RW      | 1: Send forwarded flow over USB, 0: Not
+USB_SEND_DEBUG  | 1        | RW      | 1: Send debug msgs over USB, 0: Not
+VIDEO_RATE      | 150      | RW      | Time in milliseconds between images of video transmission
+VIDEO_ONLY      | 0        | RW      | 1: High resolution video mode is ON, 0: OFF
+
+
+
+## Modes
+
+### VIDEO ONLY Mode {#VIDEO_ONLY}
+
+Set `VIDEO_ONLY` to 1 for high resolution mode. In this mode the camera image is transmitted at a higher resolution to ease the lens focus process. No flow values are calculated in this mode.
+
+###  Low Light Mode
+If `IMAGE_L_LIGHT` is set to 1, the sensor operates in low light mode.
+
+> **Note** This mode is under construction, and results are more noisy
+
+###  Test Pattern Mode
+
+If the parameter IMAGE_TEST_PAT is set to 1, the sensor inserts a vertical gray-shaded test pattern in the signal chain. 
+* Test Pattern 64x64 (VIDEO ONLY Mode is OFF)
+  ![test_pattern_64x64](../../assets/hardware/sensors/px4flow/test_pattern_64x64.png)
+* Test Pattern 376x240 (VIDEO ONLY Mode is ON)
+  ![test_pattern_376x240](../../assets/hardware/sensors/px4flow/test_pattern_376x240.png)
+
+
+## General Troubleshooting
```

Comment:
Should this go up in the user guide section?

Reply:
Yes it can be in the user guide

Reply:
Thanks! Moved in https://github.com/PX4/px4_user_guide/pull/286/commits/4307920f6c371efb71cdd05c1e65be0a509ed738